tsconfig.base.json

```diff
@@ -0,0 +1,55 @@
+{
+  "compilerOptions": {
+    "target": "es2016",
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
+    "module": "commonjs",
+    "baseUrl": ".",
+    "paths": {
+      "@dmr.is/logging": [
+        "libs/logging/src/index.ts"
+      ],
+      "@dmr.is/apm": [
+        "libs/apm/src/index.ts"
+      ],
+      "@island.is/island-ui/theme": [
+        "libs/island-ui/theme/src/index.ts"
+      ],
+      "@island.is/island-ui/utils": [
+        "libs/island-ui/utils/src/index.ts"
+      ],
+      "@island.is/island-ui/vanilla-extract-utils": [
+        "libs/island-ui/vanilla-extract-utils/src/index.ts"
+      ],
+      "@island.is/island-ui/contentful": [
+        "libs/island-ui/contentful/src/index.ts"
+      ],
```

Comment:
Do you need this? Afaik these depend on island.is contentful models.

Reply:
Good point definetly do not need this.